src/Magnum/Audio/Test/ContextTest.cpp

```diff
@@ -56,6 +60,18 @@ void ContextTest::isExtensionEnabled() {
     CORRADE_VERIFY(Context::current()->isExtensionSupported<Extensions::ALC::EXT::ENUMERATION>());
 }
 
+void ContextTest::hrtfStatus() {
+    std::ostringstream out;
+    Debug(&out) << Context::HrtfStatus::Denied;
+    CORRADE_COMPARE(out.str(), "Audio::Context::HrtfStatus::Denied\n");
+}
+
+void ContextTest::hrtfs() {
+    /* At least test that the hrtf status is `Disabled` */
+    CORRADE_VERIFY(!_context.isHrtfEnabled());
```

Comment:
I am having second thoughs on this, since this test relies on user configuration. This will work for the CIs, but I have hrtfs enabled for example, which is why tests fail. I believe this test should be removed. @mosra Would you agree?